examples/image_classification.ipynb

```diff
@@ -910,10 +910,10 @@
     }
```

Comment:
<p>Update to paragraph: </p><p> </p><p>As we can see, it does not only show the class label with the highest probability, but does return the top 5 labels, with their corresponding scores. Note that the pipelines also work with local models and image processors:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>